src/main/java/lu/oop/server/api/controllers/MessageController.java

```diff
@@ -0,0 +1,69 @@
+package lu.oop.server.api.controllers;
+
+import lu.oop.server.app.services.IMessageService;
+import lu.oop.server.app.models.messages.*;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Set;
+
+
+@RequestMapping("/messages")
+@RestController
+public class MessageController {
+
+    private IMessageService messageService;
+
+    public static class MessageRequest {
+        private String text;
+        private int senderId;
+        private int receiverId;
+
+        public String getText(){
+            return text;
+        }
+        public int getSenderId(){
+            return senderId;
+        }
+        public int getReceiverId(){
+            return receiverId;
+        }
+    }
+
+    @Autowired
+    MessageController(IMessageService MessageService) {
+        this.messageService = MessageService;
+    }
+
+    @PostMapping("/")
+    public ResponseEntity<IMessageModel> createMessage(@RequestBody MessageRequest messageRequest) {
+        try {
+            String text = messageRequest.getText();
```

Comment:
We need to find a schema validator library for spring, currently giving the api text: null will result in 500 sys error( because null is a valid String value but not a valid database entry value for the text field).